### PR TITLE
Coordinator details map moved to Hazelcast coordination strategy

### PR DESCRIPTION
**Pull request by a5anka**

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextStore.java [hunk 1]
```diff
@@ -527,7 +527,15 @@ void createSlot(long startMessageId, long endMessageId, String storageQueueName,
...
-    void updateNodeHeartbeat(String nodeId) throws AndesException;
+    boolean updateNodeHeartbeat(String nodeId) throws AndesException;
+
+    /**
+     * Update Node heartbeat value to current time
+     *
+     * @param nodeId      local node ID
+     * @param nodeAddress Hazelcast bind address of the node
+     */
+    void createNodeHeartbeatEntry(String nodeId, InetSocketAddress nodeAddress) throws AndesException;
```
Comment (abeykoon) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextStore.java]: We need to have @throws for method comments.

Comment (a5anka) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextStore.java]: :+1:

File: modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterManager.java [hunk 2]
```diff
@@ -268,29 +267,11 @@ public void localNodeElectedAsCoordinator() {
...
-        return StringUtils.EMPTY;
-    }
-
-    /**
      * Gets address of all the members in the cluster. i.e address:port
      *
      * @return A list of address of the nodes in a cluster
      */
-    public List<String> getAllClusterNodeAddresses() {
+    public List<String> getAllClusterNodeAddresses() throws AndesException {
```
Comment (abeykoon) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterManager.java]: Under method comment better to describe @return specifically mentioning what information is returned.

Comment (a5anka) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterManager.java]: Added the description to method comment

**Pull request merged**